core/src/main/java/bisq/core/grpc/CoreWalletsService.java

```diff
@@ -65,8 +65,7 @@ public long getAvailableBalance() {
         if (!walletsManager.areWalletsAvailable())
             throw new IllegalStateException("wallet is not yet available");
```

Comment:
Following the same rationale, `verifyWalletsAreAvailable` should be created.

Reply:
Makes sense, this change will be added to the latest PR in the current (and long) chain of PRs: [4329](https://github.com/bisq-network/bisq/pull/4329).

Reply:
The requested change is in commit e1fddfa, as part of PR [4329](https://github.com/bisq-network/bisq/pull/4329).